### PR TITLE
RavenDB-15924 Wrong query results or exception when querying for quot…

### PR DESCRIPTION
**Pull request by ayende**
…es or/and slashes

### CODE REVIEW

File: src/Raven.Server/Documents/Queries/QueryBuilder.cs [hunk 1]
```diff
@@ -34,6 +34,7 @@ namespace Raven.Server.Documents.Queries
     public static class QueryBuilder
     {
         private static readonly KeywordAnalyzer KeywordAnalyzer = new KeywordAnalyzer();
+        private static readonly char[] WhiteSpaceCharacters = new[]{' ', '\t'};
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Queries/QueryBuilder.cs]: is this used?

**Review by ppekrol — COMMENTED** [review]
Do we really want to introduce those changes to 4.2 at this stage?

File: src/Raven.Server/Documents/Queries/QueryBuilder.cs (hunk 1 above)
Comment (ayende) [reply src/Raven.Server/Documents/Queries/QueryBuilder.cs]: No, removed.

**Comment by ayende** [comment]
I'm fine with making this with 5.0 / 5.1 instead.

**Review by arekpalinski — APPROVED** [review]
I don't see anything wrong here. Although it's very difficult to spot a bug during a review in that piece of code. I trust our test suite but that's potentially risky change IMO.

**Pull request merged**